modules/auxiliary/scanner/smb/smb_version.rb

```diff
@@ -10,6 +10,8 @@ class MetasploitModule < Msf::Auxiliary
   # Exploit mixins should be called first
   include Msf::Exploit::Remote::DCERPC
   include Msf::Exploit::Remote::SMB::Client
+  include Msf::Exploit::Remote::Kerberos::Ticket::Storage
+  include Msf::Exploit::Remote::Kerberos::ServiceAuthenticator::Options
```

Comment:
Potentially this needs to be moved into `SMB::Client`; But we'd require a run through of modules to verify that behavior as we've only tested modules which rely on `SMB::Client::Authenticated`